src/app/view/components/dot-container-selector/dot-container-selector.component.scss

```diff
@@ -18,12 +17,19 @@
     }
 }
 
+.dot-layout-designer__grid-guides .container-selector {
```

Comment:
This css doesn't belong here... imagine when this component is being use in another place.

Reply:
done